app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -222,6 +228,16 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener {
         settings.setOnClickListener { launchSettings() }
         toggle_photo_video.setOnClickListener { handleTogglePhotoVideo() }
         change_resolution.setOnClickListener { mPreview?.showChangeResolutionDialog() }
+        countdown_toggle.setOnClickListener { toggleCountdownTimer() }
+
+        // TODO
+        // - Right now this is somewhat hard-coded
+        // - May want to look into using Android Spinner instead (similar to dropdowns)
+        // - The 3 lines below should be 1 that takes the value from front end to set the delay
+        // - https://developer.android.com/guide/topics/ui/controls/spinner
```

Comment:
Great comment here!

Reply:
Are we doing this or not? If not, remove this comment